### DISCUSSION TITLE
how to turn off text highlighting in vs code?

### DISCUSSION THREAD

**Discussion by abdulholiq-ahmad** [community/community#139587] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

how to turn off text highlighting in vs code?

**Comment by Rishikesh1601** [comment]
To turn off text highlighting in Visual Studio Code (VS Code), you can adjust various settings depending on what type of highlighting you want to disable (e.g., word highlighting, selection highlighting, or syntax highlighting). Here are the common cases and how to turn them off:

### 1. **Disable Word Highlighting**
   VS Code automatically highlights all instances of the selected word. To disable this:

   - Open **Settings** (`Ctrl + ,` or `Cmd + ,` on macOS).
   - Search for `Editor: Occurrences Highlight`.
   - Uncheck the option **Editor: Occurrences Highlight** to turn off the word highlighting.

   Alternatively, you can directly modify the `settings.json` file:
   - Go to **File > Preferences > Settings**.
   - Click on the **Open Settings (JSON)** icon at the top-right corner.
   - Add the following setting:
     ```json
     "editor.occurrencesHighlight": false
     ```

### 2. **Disable Selection Highlighting**
   VS Code highlights all instances of selected text. To turn this off:

   - Open **Settings**.
   - Search for `Editor: Selection Highlight`.
   - Uncheck the option **Editor: Selection Highlight**.

   Alternatively, in `settings.json`:
   ```json
   "editor.selectionHighlight": false
   ```

### 3. **Disable Syntax Highlighting**
   If you want to turn off syntax highlighting completely:

   - You can change the language mode of the file to "Plain Text".
     - Open the Command Palette (`Ctrl + Shift + P` or `Cmd + Shift + P` on macOS).
     - Search for `Change Language Mode`.
     - Select `Plain Text` to turn off syntax highlighting for the current file.

### 4. **Disable Bracket Matching Highlighting**
   If you want to turn off the highlight around matching brackets:

   - Open **Settings**.
   - Search for `Editor: Match Brackets`.
   - Uncheck the option **Editor: Match Brackets**.

   In `settings.json`:
   ```json
   "editor.matchBrackets": false
   ```

These steps should help you turn off any form of highlighting you want to disable in VS Code!
